crypto/ecdh_extra/ecdh_test.cc

```diff
@@ -130,12 +130,19 @@ TEST(ECDHTest, TestVectors) {
   });
 }
 
-static int has_uint128_and_not_small() {
+// Returns 1 if the curve defined by |nid| is using Montgomery representation
```

Comment:
The comment in https://github.com/aws/aws-lc/blob/adea0dbcc26b9523e45dcdbe22ea641747734cc5/crypto/ecdh_extra/ecdh_test.cc#L180-L185 needs to be adjusted:

    // The following call converts the point to Montgomery form for P-256/384.
    // For P-224, when the functions from simple.c are used, i.e. when
    // group->meth = EC_GFp_nistp224_method, the coordinate representation
    // is not changed. This is determined based on compilation flags in ec.c
    // that are duplicated below in is_curve_using_mont_felem_impl().
    // For P-521, the plain non-Motgomery representation is always used also except for 
    // the flag used in the same function.

Or feel free to edit it to make it clearer.